src/main/kotlin/bitbucket/BitbucketClient.kt

```diff
@@ -4,6 +4,8 @@ import bitbucket.data.Approve
 import bitbucket.data.PR
 import bitbucket.data.PagedResponse
 import bitbucket.data.SimpleUser
+import bitbucket.data.merge.MergeStatus
```

Comment:
There is a strange thing: I have no **bitbucket.data.merge** package in the branch.